### PR TITLE
Log ffmpeg output to separate files

### PR DESCRIPTION
**Pull request by kirtan-shah**

### CODE REVIEW

**Review by ntlhui — CHANGES_REQUESTED** [review]

File: DataServer/server.py [hunk 1]
```diff
@@ -2,6 +2,8 @@
 import datetime as dt
 import logging
 import os
+import sys
+import appdirs
 import pathlib
 import socketserver
 import uuid
```
Comment (ntlhui) [inline DataServer/server.py:9]: Sort Imports

File: DataServer/server.py [hunk 2]
```diff
@@ -86,6 +88,12 @@ def __init__(self, device_tree: devices.DeviceTree, reader: StreamReader,
         self._config = config
 
         self.hasClient = asyncio.Event()
+
+        if os.getuid() == 0:
+            self.ff_log_dir = pathlib.Path('var', 'log', 'ffmpeg_logs').absolute()
+        else:
+            self.ff_log_dir = pathlib.Path(appdirs.user_log_dir('ASMDataServer'), 'ffmpeg_logs')
```
Comment (ntlhui) [inline DataServer/server.py:95]: Shouldn't this be absolute as well?

File: DataServer/server.py [hunk 3]
```diff
@@ -183,16 +193,23 @@ async def runRTPServer(self, port: int):
         file_path = os.path.abspath(os.path.join(data_dir, device_path, fname))
         file_dir = os.path.dirname(file_path)
         pathlib.Path(file_dir).mkdir(parents=True, exist_ok=True)
-        
+
+        ff_stats_path = pathlib.Path(self.ff_log_dir, device_path, "stats.log")
+        ff_info_path = pathlib.Path(self.ff_log_dir, device_path, "info.log")
+        script_path = "-m ASM_utils.ffmpeg.split_log"
```
Comment (ntlhui) [inline DataServer/server.py:199]: This isn't a path

File: DataServer/server.py [hunk 4]
```diff
@@ -183,16 +193,23 @@ async def runRTPServer(self, port: int):
...
+               f'-reset_timestamps 1 {file_path} '
+               f' 2>&1 | {sys.executable} {script_path} {ff_stats_path} {ff_info_path}'
+            )
         proc_out = asyncio.subprocess.PIPE
         proc_err = asyncio.subprocess.PIPE
         self._log.info(f'Started ffmpeg with command: {cmd}')
         proc = await asyncio.create_subprocess_shell(cmd, stdout=proc_out,
                                                      stderr=proc_err)
         self._log.info(f'RTP Server on port {port} started outputting to {file_dir}')
+        self._log.info(f"FFmpeg logging to: {os.path.join(self.ff_log_dir, device_path)}")
```
Comment (ntlhui) [inline DataServer/server.py:212]: Why not simply use `ff_stats_path.parent`?

**Review by ntlhui — CHANGES_REQUESTED** [review]

File: DataServer/server.py (hunk 4 above)
Comment (kirtan-shah) [reply DataServer/server.py:212]: ff_log_dir already exists and is used to create ff_stats_path

**Review by ntlhui — APPROVED** [review]

**Pull request merged**